client/src/component/Dashboard/Dashboards/Bootcamp/BootcampDashboard.jsx

```diff
@@ -2,11 +2,15 @@ import React, { Component, Fragment } from 'react';
 import HackerNews from '../../../NewsFeed/HackerNews/'
 import TechNews from '../../../NewsFeed/TechNews/';
 import BootcampResourcePost from './BootcampResourcePost';
+import CompanyProfile from "../Company/views/CompanyProfile";
+import UserProfile from "./views/BootcampProfile";
 
 class BootcampDashboard extends Component {
   render() {
     return (
       <Fragment>
+        <UserProfile />
+        <CompanyProfile />
```

Comment:
I'm not sure we need CompanyProfile on the bootcamp user's dashboard. Unless this was for testing purposes?